projects/netgrif-components/src/lib/data-fields/file-list-field/file-list-default-field/file-list-default-field.component.scss

```
@@ -32,7 +32,7 @@
 
 .form-input {
     background: #FFFFFF;
-    color: #64748B;
+    //color: #64748B;
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)